cosmo_tester/resources/infrastructure_blueprints/aws/vm.yaml

```diff
@@ -24,6 +24,9 @@ inputs:
   use_net:
     default: "1"
 
+  volume_attachment_device:
```

Comment:
Given that this is ignored by AWS/the image we use/something else, it might be worth just adding a comment that this is only provided because it is required by the plugin, but isn't actually meaningful.

Reply:
Switching to reviewing as one block, not by commit- will re-raise comments as required.